CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 <a name="0.18.0"></a>
 # [0.18.0](https://github.com/ipld/js-ipld/compare/v0.17.4...v0.18.0) (2018-10-12)
 
+### BREAKING CHANGES
+
+* The API of ipld-dag-cbor changed. Links are no longer represented as JSON
+(`{"/": "base-encoded-cid"}`), but as a CID object.
+
+Example:
 
+    ipld.put({link: new CID('base-encoded-cid')}, {format: 'dag-cbor'}, (err, cid) => {…})
```

Comment:
* Does the existing format still work for `put`?
    * If so, is the plan to continue supporting the existing format into the future or remove at some point?
* Does this also affect response from a `ipld.get`?

Reply:
Suggestion: add a js code block for some syntax highlighting?

Reply:
>     * Does the existing format still work for `put`?
>       
>       * If so, is the plan to continue supporting the existing format into the future or remove at some point?

It does support the existing format for `put`. I've created https://github.com/ipld/js-ipld-dag-cbor/issues/84 to track the removal of that support.

>     * Does this also affect response from a `ipld.get`?

Yes it does, I'll add information about that.